app/views/unloading_permission/components/packages_goods/goods_item.scala.xml

```diff
@@ -0,0 +1,73 @@
+@(permission: viewmodels.PermissionToStartUnloading)
+
+@if(permission.goodsItems.head.packages.toList.length > 0) {
+<fo:table width="100%" table-layout="fixed">
+    <fo:table-column column-width="20mm"/>
+    <fo:table-column column-width="25mm"/>
+    <fo:table-column column-width="15mm"/>
+    <fo:table-column column-width="25mm"/>
+    <fo:table-column column-width="25mm"/>
+    <fo:table-body>
+        <fo:table-row height="15mm" padding-top="1mm">
+            <fo:table-cell font-weight="bold" wrap-option="wrap">
+                <fo:block>
+                    <!--Marks and numbers-->
+                    @permission.goodsItems.head.packages.toList.head.marksAndNumbersValue.map( y => y)
+                </fo:block>
+            </fo:table-cell>
+            <fo:table-cell font-weight="bold" padding-left="3mm" wrap-option="wrap">
+                <!-- Container No(s) -->
+                <fo:block>
+                    @permission.goodsItems.head.containers.head
```

Comment:
Noticed we are only grabbing the head here, do multiple goods items need to be implemented or are we going to do that in another iteration?

Reply:
Yeah, multiple goods items are on another page I think. We only use goods_item template when there is only one item. Multiple goods will be in another PR